src/search/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SearchBar from './headerBar';
+import Result from './result';
+
+const navigationOptions = {
+  header: () => <SearchBar />,
```

Comment:
do you really need to pass a method as header? I dont think so, pls double check

Reply:
The official doc add custom component as header in this way. [Here.](https://reactnavigation.org/docs/en/headers.html#docsNav)